tox.ini

```diff
@@ -36,7 +35,7 @@ deps =
 
     # The oldestdeps factor is intended to be used to install the oldest versions of all
     # dependencies that have a minimum version.
-    oldestdeps: numpy==1.16.*
+    oldestdeps: numpy==1.17.*
```

Comment:
Please don't bump the minversion here. This is out of scope.

Reply:
Done